### PR TITLE
Fixes #31902 - update pulp rpm for katello 3.18, 3.17

### PR DESCRIPTION
**Pull request by jjeffers**
This PR bumps pulp_rpm and updates VCR cassettes.

### CODE REVIEW

File: katello.gemspec [hunk 1]
```diff
@@ -55,7 +55,7 @@ Gem::Specification.new do |gem|
   gem.add_dependency "pulp_ansible_client", ">= 0.2", "< 0.5"
   gem.add_dependency "pulp_container_client", ">= 2.0.0", "< 2.2.0"
   gem.add_dependency "pulp_deb_client", ">= 2.6.0", "< 2.8.0"
-  gem.add_dependency "pulp_rpm_client", ">=3.6.2", "< 3.8.0"
+  gem.add_dependency "pulp_rpm_client", ">=3.9.0", "< 3.9.1"
```
Comment (jlsherrill) [inline katello.gemspec:58]: i think this could be < 3.10.0

Comment (jlsherrill) [reply katello.gemspec:58]: or should be, i mean

**Comment by jlsherrill** [comment]
i bet my change for the corruption workflow invalidated your vcr cassettes, although i'm not sure why there wasn't a conflict?

**Comment by jlsherrill** [comment]
[test katello]

**Comment by jjeffers** [comment]
@jturel we're green, so ready to merge if you approve

**Review by jturel — APPROVED** [review]
I see no issues here :+1:

**Pull request merged**